e2e/tests/features/journeys/registration/nhsSignupJourneyTest.e2e.ts

```diff
@@ -0,0 +1,22 @@
+import { test } from "../../../../hooks/CustomFixtures";
+
```

Comment:
This needs built out a little, there is no check if the errors come up, no checks if the first name and last name are in the input field as expected, etc

Reply:
try and cover every possible user action